apps/src/sites/studio/pages/lessons/show.js

```diff
@@ -39,6 +39,12 @@ $(document).ready(function() {
         // React key
         tip.key = _.uniqueId();
       });
+
+      activitySection.scriptLevels.forEach(scriptLevel => {
```

Comment:
This looks like it would impact the teacher facing lesson plan?

Reply:
Yes - sorry not to explain this. the Activity components is used both on the edit and show pages, and it was using scriptLevel.position to populate the bubble numbers. position already had the right bubble number on the teacher facing lesson plan, but on the lesson edit page position was relative to the activity section, and levelNumber was the value we needed to use. So, I made the show and edit pages a bit more similar by using levelNumber to be the bubble number (position relative to the beginning of the lesson) in both places.